edb/api/errors.txt

```diff
@@ -138,6 +138,9 @@
 
 0x_FF_00_00_00   ClientError
 0x_FF_01_00_00   ClientConnectionError
+0x_FF_01_01_00   ClientConnectionFailedError
+0x_FF_01_01_01   ClientConnectionFailedTemporarilyError  #SHOULD_RECONNECT
```

Comment:
Paul, last bit of bikeshedding here, I promise! Should we call this TRY_RECONNECT? Also, should we adopt enum-like syntax and use CamelCase for tags?

cc @elprans

Reply:
"try reconnect" sounds like a method name. I doesn't sound like an actionable tag for error. But we can review tag names again when we have `retry`. I'm not sure what the full list of tags we need.
 
Both in python and javascript error tags are constants, not types. In python, even if that would be an enum tag names would be snake case. If in Rust they will be types we will turn snake case into a camel case. Also I like snake case here since it's clear how to turn it into camel case without ambiguities (as there is no temptation to write `AsyncIOSomething` which is `async_i_o_something` or `async_io_something` or `asyncio_something`, so other way around it might be ambiguous)

Reply:
+1 for constant-style names.  Don't have a strong opinion on `TRY` vs `SHOULD`, but the latter sounds better to me.